draft-ietf-scitt-architecture.md

```diff
@@ -367,7 +368,8 @@ Envelope:
 : metadata, created by the Issuer to produce a Signed Statement.
 The Envelope contains the identity of the Issuer and information about the Artifact, enabling Transparency Service Registration Policies to validate the Signed Statement.
 A Signed Statement is a COSE Envelope wrapped around a Statement, binding the metadata in the Envelope to the Statement.
-In COSE, an Envelope consists of a protected header (included in the Issuer's signature) and an unprotected header (not included in the Issuer's signature).
+In COSE, an Envelope consists of a protected header (included in the Issuer's signature) and an unprotected header (not included in the Issuer's signature) and the Payload.
+The Payload may contain the contents of the Statement, or a hash using {{-COSE-HASH}}.
 
```

Comment:
Should nay not be MAY per BCP14 or was it an intentional choice to cap it lowercase?